src/mark.js

```diff
@@ -61,6 +61,7 @@ class Mark { // eslint-disable-line no-unused-vars
             "caseSensitive": false,
             "ignoreJoiners": false,
             "ignoreGroups": 0,
+            "invalidMax": true,
```

Comment:
I think it'd be okay to remove it, if the debug log shows that is was automatically corrected. What do you think?